lib/vdsm/storage/lvm.py

```diff
@@ -1486,20 +1486,14 @@ def _checkpvsblksize(pvs, vgBlkSize=None):
             raise se.VolumeGroupBlockSizeError(vgBlkSize, pvBlkSize)
```

Comment:
Commit message need update

- we did not add reason argument.
- we simplified listPVNames() - need to explain why the previous implementation was wrong. This also worth its own commit to to keep changes in real code minimal.
- we removed the code raising if we get no pv, need to explain why this is not needed. I think it can be part of the commit simplifying listPVNames(), if I remember the reason corectly.